default-recommendations/for-loop-shortcuts-test.rkt

```diff
@@ -909,3 +909,73 @@ test: "(unless ...) in a for* loop refactored to #:when clause"
        #:unless (number? x))
   (displayln x))
 ------------------------------------------------------------
+
+
+test: "for/vector with in-range 0 n gets #:length n"
+------------------------------------------------------------
+(define n 5)
+(for/vector ([i (in-range 0 n)]) i)
+------------------------------------------------------------
+------------------------------------------------------------
+(define n 5)
+(for/vector #:length n
+            ([i (in-range 0 n)])
+  i)
+------------------------------------------------------------
+
+
+test: "for/vector with in-range n gets #:length n"
+------------------------------------------------------------
+(define n 5)
+(for/vector ([i (in-range n)]) i)
+------------------------------------------------------------
+------------------------------------------------------------
+(define n 5)
+(for/vector #:length n
+            ([i (in-range 0 n)])
+  i)
+------------------------------------------------------------
+
+
+test: "for*/vector with in-range 0 n gets #:length n"
+------------------------------------------------------------
+(define n 5)
+(for*/vector ([i (in-range 0 n)]) i)
+------------------------------------------------------------
+------------------------------------------------------------
+(define n 5)
+(for*/vector #:length n
+             ([i (in-range 0 n)])
+  i)
+------------------------------------------------------------
+
+
+test: "for/vector with literal end not refactorable"
+------------------------------------------------------------
+(for/vector ([i (in-range 0 10)]) i)
+------------------------------------------------------------
```

Comment:
Since this is only one line of code, it should be written as a single-line code block instead of a multiline code block. See the [Resyntax testing framework documentation](https://docs.racket-lang.org/resyntax/Testing_Refactoring_Rules.html#%28part._.Code_.Blocks%29) for details.